AnnoyancesFilter/sections/cookies_whitelist.txt

```diff
@@ -1575,3 +1575,4 @@ lg.com#@#.cookie-check
 mycard2go.com#@#.cookiebar
 fashionette.de#@##cookies-notice
 foodnetwork.co.uk#@#.cookie-message:not(html):not(body)
+@@||bancodesio.it/sites/all/modules/eu_cookie_compliance/$~third-party
```

Comment:
`@@||bancodesio.it/sites/all/modules/eu_cookie_compliance/js/jquery.cookie-` is enough. `~third-party` is redundant in this case.

Reply:
ok